modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,9 +7,14 @@
 = Using Data Mover for CSI snapshots
 
 :FeatureName: Data Mover for CSI snapshots
-include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
+The Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs.
+
+Volume snapshot backups (VSBs) are queued using the VolumeSnapshotMover. The VolumeSnapshotMover's performance is improved by specifying a concurrent number of volumeSnapshotBackups and volumeSnapshotRestores simultaneously in Progress. The `pruneInterval` feature repacks data to free space, by determining the number of days between running restic prune on the repository.
```

Comment:
pruneInterval is specified below in the new features - should it also be detailed here? As there are several other new features not mentioned here. Just a thought

Reply:
@weshayutin should we remove the PruneInterval mention? If we leave in the PruneInterval mention, is there anything else we should also include?